src/components/widgets/DoItYourself.vue

```diff
@@ -157,6 +167,7 @@ const defaultOptions = {
 document.addEventListener('DOMContentLoaded', () => {
   // Your code here
 });`,
+  inheritCockpitStyles: false,
```

Comment:
I think this should be on by default, since we want to encourage people to match the application styling.

It's ok if existing widgets don't make use of it for backwards compatibility, but if it's necessary to make them use it if we want it enabled by default for new ones then I think that's a worthwhile tradeoff - it will make most widgets look better, and if someone doesn't like the update they'll likely try to edit it when they notice it looks different, at which point they'll be able to turn off the Cockpit styling inheritance if they don't want it.

Reply:
I don't like the idea of auto-selecting it since it's hard for the user to overcome the default styling if they are inheriting cockpit ones, which will lead to frustrations like "why my color isn't being applied?".

Reply:
As discussed in the meeting @ES-Alexander I implemented your suggestion of the custom CSS to have more priority than the inherited ones and it worked! Pushed the update already. Now they are beautiful by default.